discord/ext/alternatives/literal_converter.py

```diff
@@ -1,10 +1,16 @@
 from typing import Literal, get_args
+from discord.ext.commands import Command
 from discord.ext.commands.errors import ConversionError
 
 _old_actual_conversion = Command._actual_conversion
 
-def _actual_conversion(self, ctx, converter, argument, param):
-    if converter is Literal:
+async def _actual_conversion(self, ctx, converter, argument, param):
+    try:
+        origin = converter.__origin__
+    except AttributeError:
+        origin = None
```

Comment:
Couldn't this use `typing.get_origin` as its already 3.8+?

Reply:
Is the lib 3.8+?

Reply:
typing.Literal is 3.8+

Reply:
nvm, changed to use it.